hackathon_site/dashboard/frontend/src/pages/Teams/Teams.tsx

```diff
@@ -2,13 +2,43 @@ import React from "react";
 // import styles from "./Teams.module.scss";
 import Header from "components/general/Header/Header";
 import Typography from "@material-ui/core/Typography";
+import Grid from "@material-ui/core/Grid";
+import styles from "pages/Teams/Teams.module.scss";
+
+import TeamCardAdmin from "components/dashboard/TeamCardAdmin/TeamCardAdmin";
+import InventorySearch from "components/inventory/InventorySearch/InventorySearch";
+import { teamsList } from "testing/mockData";
+
+// import TeamCardAdmin from "../../components/dashboard/TeamCardAdmin/TeamCardAdmin";
 
 const Teams = () => {
+    const CardComponents = teamsList.map((team) => (
+        <Grid
+            item
+            md={3}
+            sm={4}
+            xs={6}
+            className={styles.dashboardGridItem}
+            key={0}
+            data-testid="team"
+            direction="column"
+            grid-template-column="true"
+        >
+            <TeamCardAdmin
+                team_name={team.TeamName}
+                members={team.Members}
+            ></TeamCardAdmin>
+        </Grid>
+    ));
+
     return (
         <>
-            <Header />
+            <Header></Header>
             <Typography variant="h1">Teams</Typography>
-            <p>IEEEEEE</p>
+            <div className={styles.searchBar}>
+                <InventorySearch></InventorySearch>
```

Comment:
you can use self-closing tag here

Reply:
Also, could you create a new search bar because this one is only used for the inventory page (it has some redux stuff in it that is not relevant to the teams page)

You can copy the inventorySearch component and delete all the unnecessary redux stuff, lmk if you need help with that!

Reply:
@ericji1326 not sure if you saw the comment, reminder to do the following^^

Reply:
changed!